camera-config/PanoCameraCalibration.yml

```diff
@@ -0,0 +1,36 @@
+%YAML:1.0
+---
+name: "hand"
+description: "Camera located on the science station for panoramas."
+# Make sure that this camera_id matches the one defined in the udev rules.
+camera_id: 100
+calib_info:
+   calibration_time: "Sat 01 May 2021 01:54:16 PM PDT"
+   board_width: 11
+   board_height: 8
+   square_size: 20.
+   flags: 0
+   avg_reprojection_error: 4.5744250014767379e-01
+intrinsic_params:
+   image_width: 640
+   image_height: 480
+   camera_matrix: !!opencv-matrix
+      rows: 3
+      cols: 3
+      dt: d
+      data: [ 6.4936764773277400e+02, 0., 3.3546152305967058e+02, 0.,
+          6.4776804982533793e+02, 2.4785660889077488e+02, 0., 0., 1. ]
+   distortion_coefficients: !!opencv-matrix
+      rows: 5
+      cols: 1
+      dt: d
+      data: [ -4.5670452858419575e-01, 2.1512832208218641e-01,
+          -1.1179480533495888e-03, -5.7450430930239835e-04, 0. ]
+extrinsic_params: !!opencv-matrix
+  rows: 4
+  cols: 4
```

Comment:
These are all unused. We should redo how we handle camera configuration, especially considering the changes in Camera.cpp